modules/users/client/config/users.authentication.client.routes.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('users')
+    .config(routeConfig);
+
+  routeConfig.$inject = ['$stateProvider'];
+
+  function routeConfig($stateProvider) {
+    $stateProvider
+      .state('authentication', {
+        abstract: true,
+        url: '/authentication',
+        views: {
+          'main@': {
+            templateUrl: 'modules/core/client/views/layouts/fixed.one.column.client.view.html',
+            controller: 'AuthenticationController'
+          }
+        }
+      })
+      .state('authentication.signup', {
+        url: '/signup',
+        views: {
+          'row-1': {
```

Comment:
@codydaig @mleanos maybe you guys can think of a better way to do this, but I don't think this is the best way.